pyproject.toml

```diff
@@ -48,6 +48,8 @@ dev = [
     "types-wtforms>=3.2.1.20250304",
     "html5lib>=1.1",
     "types-html5lib>=1.1.11.20241018",
+    "types-pytz>=2025.2.0.20250326",
+    "freezegun>=1.5.1",
```

Comment:
I might just be missing something but does this PR consume this new dependency?

Reply:
nope 🫠  I started going down that path and then stopped when I hit the db timestamp issue. Let me rip it out.